backend/src/agents/tools/browser_tools.py

```
@@ -0,0 +1,690 @@
+    name: str
+    description: str
+    args_schema: type[BaseModel] = TakeScreenshotSchema
+    page: Page
+
+    async def _run(self, ss_name: str, full_page: bool) -> str:
+        try:
+            screenshot_options = {
+                "full_page": full_page,
+                "path": ss_name if ss_name.endswith('.png') else f"{ss_name}.png"
+            }
+            screenshot_bytes = await self.page.screenshot(**screenshot_options)
```

Ruff (0.12.2)

Local variable `screenshot_bytes` is assigned to but never used

Remove assignment to unused variable `screenshot_bytes`

(F841)